voucherApp/src/main/java/com/mountain/voucherApp/adapter/in/console/ConsoleController.java

```diff
@@ -0,0 +1,91 @@
+package com.mountain.voucherApp.adapter.in.console;
+
+import com.mountain.voucherApp.adapter.in.VoucherAppController;
+import com.mountain.voucherApp.application.port.in.VoucherAppUseCase;
+import com.mountain.voucherApp.application.port.in.VoucherCreateDto;
+import com.mountain.voucherApp.application.port.in.VoucherIdUpdateDto;
+import com.mountain.voucherApp.shared.enums.DiscountPolicy;
+import com.mountain.voucherApp.shared.io.InputConsole;
+import com.mountain.voucherApp.shared.io.OutputConsole;
+import com.mountain.voucherApp.shared.utils.DiscountPolicyUtil;
+import com.mountain.voucherApp.domain.Voucher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+
+import java.util.UUID;
+
+import static com.mountain.voucherApp.shared.constants.ProgramMessage.*;
+
+@Controller
+public class ConsoleController implements VoucherAppController {
+
+    private static final Logger log = LoggerFactory.getLogger(ConsoleController.class);
+    private static final int NOT_EXIST_DATA_NUMBER = -1;
+    private final InputConsole inputConsole;
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(ConsoleController.class);
    private static final int NOT_EXIST_DATA_NUMBER = -1;
    
    private final InputConsole inputConsole;
```

개행으로 상수와 필드를 구분하면 더 좋습니다 :)